README.md

```diff
@@ -12,6 +12,54 @@ The app design, UX and implementation was done by [Ubique](https://www.ubique.ch
 This service provides an API, which is consumed by other CovidCertificate backend services. It offers an endpoint to allow the decoding and verification of a given Covid Certificate.
 It regularly sends a request to the [Verifier Service](https://github.com/admin-ch/CovidCertificate-App-Verifier-Service) to keep an up-to-date list of trusted DSCs with which Covid Certificates could be signed.
 
+## Simple Verification
+
+In order to allow the independent usage of the Verification-Check-Service without deeper knowledge of the source code, the repository includes a Dockerfile to start the service in a separate container. However, one needs to be in possession of the Verifier-Service's baseurl and a corresponding api-token to use the service.
```

Comment:
Base URLs sind bekannt, bitte hier auflisten:
* ABN: https://www.cc-a.bit.admin.ch
* DEV: https://www.cc-d.bit.admin.ch
* PROD: https://www.cc.bit.admin.ch

Reply:
See Felix's comment regarding setting the baseurl based on the profile